docs/api/geometries/EdgesGeometry.html

```diff
@@ -14,40 +14,23 @@
 
 		<div class="desc">This can be used as a helper object to view the edges of a [page:Geometry Geometry] object.</div>
 
-		<!-- <iframe id="scene" src="scenes/geometry-browser.html#EdgeGeometry"></iframe>
-
-		<script>
-
-		// iOS iframe auto-resize workaround
-
-		if ( /(iPad|iPhone|iPod)/g.test( navigator.userAgent ) ) {
-
-			var scene = document.getElementById( 'scene' );
-
-			scene.style.width = getComputedStyle( scene ).width;
-			scene.style.height = getComputedStyle( scene ).height;
-			scene.setAttribute( 'scrolling', 'no' );
-
-		}
-
-		</script> -->
-
 		<h2>Example</h2>
 
 		[example:webgl_helpers helpers]
 
 		<code>
 var geometry = new THREE.BoxBufferGeometry( 100, 100, 100 );
 var edges = new THREE.EdgesGeometry( geometry );
-var line = new THREE.LineSegments( edges );
+var line = new THREE.LineSegments( edges, new THREE.LineBasicMaterial( { color: 0xffffff } ) );
 scene.add( line );
 		</code>
 
 		<h2>Constructor</h2>
 
-		<h3>[name]( [page:Geometry geometry] )</h3>
+		<h3>[name]( [page:Geometry geometry], [page:Integer thresholdAngle] )</h3>
 		<div>
-		geometry — any geometry object.
+		geometry — Any geometry object.<br />
+		thresholdAngle — A valid edge is only detected if the angle between two consecutive faces is greater than this value, default = 1.
```

Comment:
thresholdAngle — An edge is only rendered if the angle (in degrees) between the face normals of the two adjoining faces exceeds this value. default = 1 degree.